aten/src/ATen/DLConvertor.cpp

```diff
@@ -153,7 +153,7 @@ DLManagedTensor* toDLPack(const Tensor& src) {
   atDLMTensor->tensor.dl_tensor.data = src.data_ptr();
   int64_t device_id = 0;
   if (src.is_cuda()) {
-    device_id = src.get_device();
+    device_id = src.device().index();
```

Comment:
Man, this is so naughty. CC @bddppq this cannot possible work with HIP at all. (No action needed, I guess. Maybe file a task if you're feeling charitable.)